## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 452426`
Commit: [`09b4831922f608bf89612c83b50a53eef7cfe24c`](https://github.com/NixOS/nixpkgs/commit/09b4831922f608bf89612c83b50a53eef7cfe24c) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/09b4831922f608bf89612c83b50a53eef7cfe24c..pull/452426/head))
Merge: [`5f7849fc232df27cbe7cc0a9e17d7f0570a142a1`](https://github.com/NixOS/nixpkgs/commit/5f7849fc232df27cbe7cc0a9e17d7f0570a142a1)

Logs: https://github.com/Sigmanificient/nixpkgs-review-gha/actions/runs/18554600108


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>cronie</li>
    <li>cronie.man</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>cronie</li>
    <li>cronie.man</li>
  </ul>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>cronie</li>
    <li>cronie.man</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>cronie</summary>
<pre>creating cron-paths.h
make  all-am
make[1]: Entering directory &#x27;/nix/build/nix-5798-2481866946/source&#x27;
  CC       src/cronnext.o
  CC       src/database.o
  CC       src/job.o
  CC       src/user.o
  CC       src/entry.o
src/entry.c:488:17: warning: passing arguments to a function without a prototype is deprecated in all versions of C and is not supported in C23 [-Wdeprecated-non-prototype]
  488 |                 (*error_func) (ecodes[(int) ecode]);
      |                               ^
1 warning generated.
  CC       src/env.o
src/env.c:147:17: error: use of undeclared identifier &#x27;environ&#x27;
  147 |         for (procenv = environ; *procenv != NULL; ++procenv) {
      |                        ^~~~~~~
1 error generated.
make[1]: *** [Makefile:749: src/env.o] Error 1
make[1]: Leaving directory &#x27;/nix/build/nix-5798-2481866946/source&#x27;
make: *** [Makefile:498: all] Error 2</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>cronie</li>
    <li>cronie.man</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>cronie</summary>
<pre>creating cron-paths.h
make  all-am
make[1]: Entering directory &#x27;/nix/build/nix-6722-588022008/source&#x27;
  CC       src/cronnext.o
  CC       src/database.o
  CC       src/job.o
  CC       src/user.o
  CC       src/entry.o
src/entry.c:488:17: warning: passing arguments to a function without a prototype is deprecated in all versions of C and is not supported in C23 [-Wdeprecated-non-prototype]
  488 |                 (*error_func) (ecodes[(int) ecode]);
      |                               ^
1 warning generated.
  CC       src/env.o
src/env.c:147:17: error: use of undeclared identifier &#x27;environ&#x27;
  147 |         for (procenv = environ; *procenv != NULL; ++procenv) {
      |                        ^~~~~~~
1 error generated.
make[1]: *** [Makefile:749: src/env.o] Error 1
make[1]: Leaving directory &#x27;/nix/build/nix-6722-588022008/source&#x27;
make: *** [Makefile:498: all] Error 2</pre>
</details>
</details>

--- a/pkgs/by-name/cr/cronie/package.nix
+++ b/pkgs/by-name/cr/cronie/package.nix
@@ -41,6 +41,6 @@ stdenv.mkDerivation (finalAttrs: {
     ];
     mainProgram = "crond";
     maintainers = [ ];
-    platforms = lib.platforms.all;
+    platforms = lib.platforms.linux;
   };
 })

--- a/pkgs/by-name/cr/cronie/package.nix
+++ b/pkgs/by-name/cr/cronie/package.nix
@@ -16,6 +16,11 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-WrzdpE9t7vWpc8QFoFs+S/HgHwsidRNmfcHp7ltSWQw=";
   };
 
+  configureFlags = [
+    "--localstatedir=/var"
+    "--sysconfdir=/etc"
+  ];
+
   nativeBuildInputs = [ autoreconfHook ];
 
   outputs = [

